### PR TITLE
Read zip files with URI encoded paths

### PR DESCRIPTION
**Pull request by olafurpg**
This PR is an attempt to fix a bug in the scalafix test suite on Windows that I have been unable to reproduce in the Scalameta Appveyor CI https://ci.appveyor.com/project/scalacenter/scalafix/build/171#L996

```
[info]   Cause: java.net.URISyntaxException: Illegal character in opaque part at index 20: jar:file:/C:/Program Files/Java/jdk1.8.0/jre/lib/resources.jar
```

A repl session shows this

```scala
@ java.net.URI.create("jar:file:/C:/Program Files/Java/jdk1.8.0/jre/lib/resources.jar")
java.lang.IllegalArgumentException: Illegal character in opaque part at index 20: jar:file:/C:/Program Files/Java/jdk1.8.0/jre/lib/resources.jar
@ java.net.URI.create("jar:file:/C:/Program%20Files/Java/jdk1.8.0/jre/lib/resources.jar")
res2: java.net.URI = jar:file:/C:/Program%20Files/Java/jdk1.8.0/jre/lib/resources.jar
@ Paths.get("U A").toUri.getPath
res3: String = "/Users/ollie/dev/scalameta/U A"
@ Paths.get("U A").toUri.toString
res4: String = "file:///Users/ollie/dev/scalameta/U%20A"
```

### CODE REVIEW

**Comment by olafurpg** [comment]
Was able to reproduce it finally!

Failure: https://ci.appveyor.com/project/olafurpg/scalameta/build/946
Success: https://ci.appveyor.com/project/olafurpg/scalameta/build/947 (after e1c4574)

File: build.sbt [hunk 1]
```diff
@@ -845,7 +848,8 @@ inScope(Global)(
     credentials ++= (for {
       username <- sys.env.get("SONATYPE_USERNAME")
       password <- sys.env.get("SONATYPE_PASSWORD")
-    } yield Credentials("Sonatype Nexus Repository Manager", "oss.sonatype.org", username, password)).toSeq,
+    } yield
+      Credentials("Sonatype Nexus Repository Manager", "oss.sonatype.org", username, password)).toSeq,
     PgpKeys.pgpPassphrase := sys.env.get("PGP_PASSPHRASE").map(_.toCharArray())
   )
 )
```
Comment (xeno-by) [inline build.sbt]: Looks like build.sbt has been inadvertently reformatted.

File: tests/jvm/src/test/scala/scala/meta/tests/metacp/VisitClasspathSuite.scala [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+import java.nio.file.FileVisitResult
+import java.nio.file.Path
+import java.nio.file.SimpleFileVisitor
+import java.nio.file.attribute.BasicFileAttributes
+
+class VisitClasspathSuite extends BaseMetacpSuite {
+
+
+}
```
Comment (xeno-by) [inline tests/jvm/src/test/scala/scala/meta/tests/metacp/VisitClasspathSuite.scala]: Is this test intended to be empty?

File: build.sbt (hunk 1 above)
Comment (olafurpg) [reply build.sbt]: Yes, that may have been caused by vscode reformatting on save 😅 I've added build.sbt to .scalafmt.conf

**Pull request merged**